pkg/security/apiserver/admission/sccadmission/scc_exec.go

```diff
@@ -25,21 +25,21 @@ func RegisterSCCExecRestrictions(plugins *admission.Plugins) {
 }
 
 var (
-	_ = admission.Interface(&sccExecRestrictions{})
 	_ = initializer.WantsAuthorizer(&sccExecRestrictions{})
 	_ = oadmission.WantsSecurityInformer(&sccExecRestrictions{})
 	_ = kadmission.WantsInternalKubeClientSet(&sccExecRestrictions{})
+	_ = admission.ValidationInterface(&sccExecRestrictions{})
 )
 
-// sccExecRestrictions is an implementation of admission.Interface which says no to a pod/exec on
+// sccExecRestrictions is an implementation of admission.ValidationInterface which says no to a pod/exec on
 // a pod that the user would not be allowed to create
 type sccExecRestrictions struct {
 	*admission.Handler
 	constraintAdmission *constraint
 	client              kclientset.Interface
 }
 
-func (d *sccExecRestrictions) Admit(a admission.Attributes) (err error) {
```

Comment:
Double check validating admission gets called on exec requests… it's not a normal storage impl

Reply:
Do we have an e2e test that checks if this is active and effective?

Reply:
`Validate` is called: https://github.com/kubernetes/kubernetes/blob/d1636b8019fa042eb1135263b00293b1c806b1d7/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go#L132

I added an integration test for SCCExecRestrictions.